Dockerfile

```diff
@@ -20,8 +20,32 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get update -yqq && \
     python3-pylibacl \
     python3-pyxattr
 
+# Build dependencies specific for rdiff-backup development and testing
+RUN DEBIAN_FRONTEND=noninteractive apt-get update -yqq && \
+    apt-get install -y --no-install-recommends \
+    tox \
+    rdiff \
+    # /usr/include/sys/acl.h is required by test builds
+    libacl1-dev \
+    # tox_slow uses rsync for comperative benchmarking
+    rsync
+
 # Build dev image
 # docker build --pull --tag rdiff-backup-dev:debian-sid .
 
-# Build rdiff-backup
-# docker run -it -v ${PWD}:/build -w /build rdiff-backup-dev:debian-sid ./setup.py build
+# Build rdiff-backup (assumes source is in directory 'rdiff-backup' and it's parent is writeable)
+# docker run -it -v ${PWD}/..:/build -w /build/rdiff-backup rdiff-backup-dev:debian-sid ./setup.py build
+
+RUN useradd -ms /bin/bash testuser
+
+# Run tests (note session will user=root inside Docker)
+# docker run -it -v ${PWD}/..:/build -w /build/rdiff-backup rdiff-backup-dev:debian-sid bash
+#   cd ..
+#   curl -LO https://github.com/ericzolf/rdiff-backup/releases/download/Testfiles2019-08-10/rdiff-backup_testfiles_2019-08-10.tar.gz
+#   tar xvf *.tar.gz # This must be run as root
+#   ./rdiff-backup_testfiles.fix.sh # This must be run as root
```

Comment:
if you run it as root directly (without sudo), you need to give the name and group of the user as parameter, e.g. `./rdiff-backup_testfiles.fix.sh testuser testuser` (the 2nd testuser is meant as group name)